workflows/diagnostics/fv3net/diagnostics/offline/views/create_report.py

```diff
@@ -396,7 +403,7 @@ def create_report(args):
         }
         wandb.init(config=wandb_config)
         wandb.log(metrics)
-
+    gsrm = loaders.BatchesLoader.from_dict(metadata["training_config"]).gsrm_name
```

Comment:
I suspect we may want to specify the gsrm name somewhere else, since we'd run into this issue again in the prognostic run diags where we don't have access to the batches object, but I'm guessing @AnnaKwa has a better idea of where exactly.

Reply:
Changed to using `ds_diags` dimension names to infer gsrm name.